paddle/framework/attribute.h

```diff
@@ -29,11 +29,13 @@ namespace framework {
 
 typedef boost::variant<boost::blank, int, float, std::string, std::vector<int>,
                        std::vector<float>, std::vector<std::string>,
-                       std::vector<std::pair<int, int>>>
+                       std::vector<std::pair<int, int>>, BlockDesc>
```

Comment:
Actually, this variant is `BlockDesc*` to avoid memcpy.

Reply:
done